lstchain/scripts/lstchain_data_create_time_calibration_file.py

```diff
@@ -0,0 +1,89 @@
+import argparse
+import numpy as np
+from traitlets.config.loader import Config
+from ctapipe.io import event_source
+from lstchain.calib.camera.r0 import LSTR0Corrections
+from lstchain.io.config import read_configuration_file
+from lstchain.calib.camera.time_correction_calculate import TimeCorrectionCalculate
+
+
+''' 
+Script to create drs4 time correction coefficients
+
+'''
+
+parser = argparse.ArgumentParser()
+
+# Required arguments
+parser.add_argument("--input_file",
+                    help="Path to fitz.fz file to create pedestal file.",
+                    type=str)
+
+parser.add_argument("--output_file",
+                    help="Path where script create pedestal file",
+                    type=str)
+
+# Optional argument
+parser.add_argument("--max_events",
+                    help="Maximum numbers of events to read. Default = 20000",
```

Comment:
is this the default number of events that you are using? Or are you using `None` to read them all?

Reply:
It is the default value, I guess @pawel21  suggested it because it is enough to get a good estimation

Reply:
ok, thanks, @pawel21 could you confirm that this is enough to get this accepted?

Reply:
@rlopezcoto , Yes, this should be enough to fill all bins in relative short time.